runtimes/typescript/lbr-plutus/build.nix

```diff
@@ -0,0 +1,15 @@
+{ ... }:
+{
+  perSystem = { config, ... }:
+    let
+      typescriptFlake =
+        config.overlayAttrs.extras.typescriptFlake {
+          name = "lbr-plutus-typescript";
+          src = ./.;
+          dependencies = [ config.packages."lbr-prelude-typescript-tarball" ];
```

Comment:
This is where we need to also specify `devShellHook` (which installs pre-commit command when in the devShell), `devShellTools` etc.